extensions/blocks/social-previews/modal.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Social Previews modal component.
+ *
+ * Shows individual previews in modal window.
```

Comment:
minor comment: Is this block doc ok? Feel free to ignore it.

Reply:
I've copied this from elsewhere in Jetpack extensions and it had the header 🤷‍♂️